async_upnp_client/utils.py

```diff
@@ -59,8 +59,20 @@ def combine(self, other: "CaseInsensitiveDict") -> "CaseInsensitiveDict":
         """
         # pylint: disable=protected-access
         _combined = CaseInsensitiveDict.__new__(CaseInsensitiveDict)
-        _combined._data = {**self._data.copy(), **other._data.copy()}
-        _combined._case_map = {**self._case_map.copy(), **other._case_map.copy()}
+        _combined._data = {**self._data, **other._data}
+        _combined._case_map = {**self._case_map, **other._case_map}
```

Comment:
I forgot to remove the `copy()` when I built this from `.copy()` so it did a double copy 🤦